### DISCUSSION TITLE
Cannot read property 'default' of null

### DISCUSSION THREAD

**Discussion by sirbh** [vercel/next.js#29545] [Help]
On step 3 after running yarn dev I am getting following on my screen

Server Error
TypeError: Cannot read property 'default' of null

`];
  113 | function isStaticRequire(src) {
> 114 |     return src.default !== undefined;
      |               ^
  115 | }
  116 | function isStaticImageData(src) {
  117 |     return src.src !== undefined;`

**Comment by rntdrts** [comment]
It happened to me because the src had a `null/undefined` value. You need to make sure the src is defined with a valid image. Empty strings also don't work.

**Reply by emekaelo** [reply]
Thanks...had to perform the check `imageProp && <Image .../>` to render only images with valid src

**Reply by muddy700** [reply]
Thanks a lot.

**Reply by ilyaskarim** [reply]
next/image should add a check inside component I have used this component in ton of places 😞

**Comment by rashap224** [comment]
"use client"
import { useKindeBrowserClient } from '@kinde-oss/kinde-auth-nextjs'
import Image from 'next/image';
import React from 'react'

function DashboardHeader() {

  const {user} = useKindeBrowserClient();
  return (
   <div>
    <Image src={user?.picture} 
    width={40}
    height={40}
    
    /> 
   </div>
  )
}

export default DashboardHeader, i m getting the same error, pls correct someone
